dive_core/data_core.cpp

```diff
@@ -772,6 +772,42 @@ void CaptureMetadataCreator::FillHardwareSpecificStates(EventStateInfo::Iterator
         event_state_it->SetEnableAllHelperLanes(enable_all_helper_lanes);
         event_state_it->SetEnablePartialHelperLanes(enable_partial_helper_lanes);
     }
+
+    // UBWC
+    // in the case of BUF_INFO and RB_DEPTH_BUFFER_INFO, A6xx and A7xx+ variants share the same
+    // register but with different bitfields, so we cannot use IsFieldEnabled() to check if the
+    // field is enabled or not for current GPU
+    uint32_t           rt_id = 0;
+    constexpr uint32_t kElemCount = 8;
+    uint32_t           rb_mrt_buf_info_reg_start = GetRegOffsetByName("RB_MRT0_BUF_INFO");
+    DIVE_ASSERT(rb_mrt_buf_info_reg_start != kInvalidRegOffset);
+    while (rt_id < 8 && m_state_tracker.IsRegSet(rb_mrt_buf_info_reg_start + kElemCount * rt_id))
+    {
+        RB_MRT_BUF_INFO rb_mrt_buf_info;
+        const uint32_t  mrt_reg = rb_mrt_buf_info_reg_start + kElemCount * rt_id;
+        rb_mrt_buf_info.u32All = m_state_tracker.GetRegValue(mrt_reg);
+        event_state_it->SetUBWCEnabled(rt_id, rb_mrt_buf_info.bitfields.COLOR_TILE_MODE == TILE6_3);
+        if (GetGPUVariantType() == kA7XX)
+        {
+            event_state_it->SetUBWCLosslessEnabled(rt_id,
```

Comment:
lossy compression is supported in UBWC3.0 which is added since A640, so in theory we could enable these for A6xx, but only enable for a7xx for now just to be consistent with the bitfield defined in turnip